### ISSUE TITLE
Document version support more accurately

### ISSUE THREAD

**Issue by alxndrsn** [pouchdb/pouchdb#8799]
Version support at https://pouchdb.com/learn.html is out of date:

![Screenshot_2023-10-19_08-22-18](https://github.com/pouchdb/pouchdb/assets/191496/bdb8c60a-5f24-4bc4-be7b-6584a87cb241)

It would be helpful to update this page, and add a table of older pouch versions and which browsers they support.

**Comment by parthrc** [comment]
Hey @alxndrsn can I work on this?

**Comment by alxndrsn** [comment]
@parthrc definitely!  I think collecting info here would probably be a good first step before updating the docs.

**Comment by parthrc** [comment]
@alxndrsn I am guessing we would have to test using something like Browserstack?
